extensions/resteasy-reactive/quarkus-resteasy-reactive/runtime/src/main/java/io/quarkus/resteasy/reactive/server/runtime/security/SecurityContextOverrideHandler.java

```diff
@@ -102,6 +102,12 @@ public Map<String, Object> getAttributes() {
                             return oldAttributes;
                         }
 
+                        @Override
+                        public List<Function<Permission, Uni<Boolean>>> getPermissionCheckers() {
+                            throw new UnsupportedOperationException(
```

Comment:
What do you think of returning an empty list, not sure users need to know the context has been replaced, if they did not write a custom augmentor adding the checkers in the first place

Reply:
Actually I don't know at all. I copied it from here https://github.com/quarkusio/quarkus/blob/main/extensions/resteasy-reactive/quarkus-resteasy-reactive/runtime/src/main/java/io/quarkus/resteasy/reactive/server/runtime/security/SecurityContextOverrideHandler.java#L70, but there was no comment and I don't know what is acceptable... It could be we can even pass there old permissions as unlike for roles, this context has nothing to do with permissions, therefore you can't modify them. But I don't want to do anything I don't understand. WDYT?

Reply:
There is no `getPermssionCheckers` now, so there is no such issue either.